azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/implementation/OfferImpl.java

```diff
@@ -7,7 +7,7 @@
 import com.microsoft.azure.management.resources.fluentcore.arm.Region;
 
 /**
- * The implementation for {@link VirtualMachineImage.Offer}.
+ * The implementation for {@link Offer}.
```

Comment:
Let me check it, created an issue https://github.com/Azure/azure-sdk-for-java/issues/863